feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouViewModel.kt

```diff
@@ -127,6 +131,17 @@ class ForYouViewModel @Inject constructor(
         if (newsResourceId == deepLinkedNewsResource.value?.id) {
             savedStateHandle[LINKED_NEWS_RESOURCE_ID] = null
         }
+        analyticsHelper.logEvent(
```

Comment:
would it be a good idea to encapsulate distinct events in an extension function?

Reply:
Good idea. In other modules we added an `AnalyticsExtensions.kt` file which contained `logX` methods specific to that module. 

Example: https://github.com/android/nowinandroid/blob/2a4d0434a5c8a794f86410850e25d66e846eb97f/core/data/src/main/java/com/google/samples/apps/nowinandroid/core/data/repository/AnalyticsExtensions.kt#L69